fix(account): update how to change the account language (#3664)
scaleway/docs-content: console/account/how-to/change-language.mdx

- A [Scaleway account](https://console.scaleway.com)

1. Log into the [Scaleway console](https://console.scaleway.com).

> 2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays.

Author: ```suggestion
2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays:
```

> <Lightbox src="scaleway-console-menu.webp" alt="" />

Author: ```suggestion
 <Lightbox src="scaleway-console-menu.webp" alt="" />
```

3. In the User information section, click the **Edit profile** button. The **Edit your account profile** screen displays:
 <Lightbox src="scaleway-edit-profile.webp" alt="" /> 
4. Choose your preferred language from the drop-down list, then click **Confirm changes**.

> Your console will now be displayed in your preferred language, and you will also receive marketing and product communications in this language. You can repeat this procedure to change your preferred language at any time.

Reviewer: ```suggestion
Your console will now be displayed in your preferred language, which will also be used for all marketing and product communications you receive. You can change your preferred language at any time by repeating this procedure.
```

- A [Scaleway account](https://console.scaleway.com)

1. Log into the [Scaleway console](https://console.scaleway.com).

> 2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays.

Author: ```suggestion
2. Click your Organization name in the top right corner of the console to open the Organization drop-down menu (A), then click the **Profile** button (B). The **User Account** screen displays.
<Lightbox src="scaleway-console-menu.webp" alt="" />
```

Author: fix(account): add image
